inc/switchtec/switchtec.h

```diff
@@ -210,11 +210,14 @@ enum switchtec_log_parse_type {
 /**
  * @brief Information about log file and log definition file
  */
-struct switchtec_log_file_ver_info {
+struct switchtec_log_file_info {
 	unsigned int log_fw_version;
 	unsigned int log_sdk_version;
 	unsigned int def_fw_version;
 	unsigned int def_sdk_version;
+	bool version_mismatch;
+	bool overflow;
+	bool rsvd[6];
```

Comment:
Do we need these reserved bools? Seems we can just expand this structure if we need to...